mapr/README.md

```diff
@@ -16,7 +16,13 @@ The MapR check is included in the [Datadog Agent][2] package but requires additi
 
 - [MapR monitoring][3] is running correctly.
 - You have an available [MapR user][4] (with name, password, UID, and GID) with the 'consume' permission on the `/var/mapr/mapr.monitoring/metricstreams` stream. This may be an already existing user or a newly created user. If you want the user to be called `dd-agent`, create it before installing the Agent.
-- You have generated a [service long-lived ticket][5] for this user that is readable by the `dd-agent` user.
+
+##### On a non secure cluster
+By default the `dd-agent` user will be used to consume from MapR so the `dd-agent` user permissions should be configured to consume from the kafka topic.
+If you wish to use a different user you can follow [this guide][12] so that the `dd-agent` user can impersonate another POSIX/MapR user.
+
+##### On a secure cluster
+You will need to generate a [service long-lived ticket][5] for this user that is readable by the `dd-agent` user.
```

Comment:
```suggestion
##### On a non secure cluster
Follow [this guide][12] so that the `dd-agent` user can impersonate this MapR user.

##### On a secure cluster
Generate a [long-lived service ticket][5] for this user that is readable by the `dd-agent` user.
```

Reply:
I like this wording much better too. Also, "non-secure" should be hyphenated.